### ISSUE TITLE
About 2D pose

### ISSUE THREAD

**Issue by MilkLoo** [hongsukchoi/3DCrowdNet_RELEASE#34]
Hello, according to the paper, I can see that the image has 2Dpose input, but in the code, I don't see where the 2Dpose input comes from. Could you please help me understand the following code?

**Comment by hongsukchoi** [comment]
Hi!

In the demo code, you can find the source here: https://github.com/hongsukchoi/3DCrowdNet_RELEASE/blob/79284d533cd88515aec08fa04ee0eb50f847c6af/demo/demo.py#L119

In the dataloader code,

you can look at the  `def load_data(self):` function.

**Comment by MilkLoo** [comment]
![7a6875956a75d2f536fbbae3224ba5f](https://github.com/hongsukchoi/3DCrowdNet_RELEASE/assets/134379440/c1a4b394-3a17-4521-9d54-a80bc8356912)
Hello, I want to ask, I use your trained weight files and pictures, demo.py, but the effect is not so good, is what I did wrong?

**Comment by hongsukchoi** [comment]
Could you visualize your 2D pose input and full error log?

**Comment by MilkLoo** [comment]
![49636e1cb7cdd9322d1b04e5ed45f92](https://github.com/hongsukchoi/3DCrowdNet_RELEASE/assets/134379440/0060f124-64b9-42ce-aa0a-91c6d3c384d1)
![2dd5b60b8a4f0e4bc761b1339519583](https://github.com/hongsukchoi/3DCrowdNet_RELEASE/assets/134379440/8d290a9e-da27-4eab-85d0-e10ae04e6c6d)
I follow the operation of Redame.md step by step, and input is also downloaded from the link.

**Comment by hongsukchoi** [comment]
I see. Which link did you use?

And I think you can solve this problem by tuning the parameter here: https://github.com/hongsukchoi/3DCrowdNet_RELEASE/blob/79284d533cd88515aec08fa04ee0eb50f847c6af/demo/demo.py#L142

The input 2D pose has an inaccurate lower body pose. You can filter those joints by increasing the threshold.

**Comment by MilkLoo** [comment]
![90566cbe2ca20c30bd953d6d245cd4d](https://github.com/hongsukchoi/3DCrowdNet_RELEASE/assets/134379440/4d7c39c4-2620-42ae-a017-995bcc5d79bc)
![b5133ae1af2b8269265b40899291c14](https://github.com/hongsukchoi/3DCrowdNet_RELEASE/assets/134379440/b7fb57fc-0ad3-4603-a402-0f13eb28639a)
Hello, I have adjusted the parameters and tried many times, but the effect is still not very good. What is to be done?

**Comment by hongsukchoi** [comment]
This looks definitively wrong. Can you show me the code that you changed?
Or did you run without any modification?

Best regards,
Hongsuk Choi
https://hongsukchoi.github.io

**Comment by MilkLoo** [comment]
I have not modified the code, is directly configured after the running code

**Comment by hongsukchoi** [comment]
I see. Then, I feel like the model checkpoint is actually not loaded. Could
you change the model load code to use ‘strict=True’ and see if there’s any
error?

Best regards,
Hongsuk Choi
https://hongsukchoi.github.io

**Comment by MilkLoo** [comment]
![985de15b5201cf3c8524f3df57a5737](https://github.com/hongsukchoi/3DCrowdNet_RELEASE/assets/134379440/5b2441b6-f655-46bc-ad20-16347bc86d5e)
![57fc7fcd34c517b05d01a9172749753](https://github.com/hongsukchoi/3DCrowdNet_RELEASE/assets/134379440/5db149b1-e66e-4375-b038-862c1703eff3)
Hello, I changed your parameter to True, but the effect is still the same.

**Comment by hongsukchoi** [comment]
Hi! It seems the code is giving random results. But I can't find a reason if you didn't change the code... I hope others people can try the current code and discuss. Unfortunately, I have no resource to run the code. Maybe you can check the inference code on 3DPW benchmark first.

**Comment by MilkLoo** [comment]
Hello, I have solved this problem with the network after retraining, thank you!

**Comment by hongsukchoi** [comment]
@MilkLoo That sounds great! Maybe there's problem in the uploaded weight? I will check it out once I get resource again.

**Comment by MilkLoo** [comment]
@hongsukchoi 

I'm really sorry, I still need to trouble you, now I have the 2D pose data of Human3.6M test set, I want to use the model to evaluate the results, but the code of the evaluation part is not good at writing, can you share it? Thank you.

**Comment by hongsukchoi** [comment]
Hi!

I am not sure I understood your question. You could change the testing dataset here: https://github.com/hongsukchoi/3DCrowdNet_RELEASE/blob/main/assets/yaml/3dpw.yml

and run the command here:
https://github.com/hongsukchoi/3DCrowdNet_RELEASE/blob/main/assets/running.md
